### ISSUE TITLE
How to install the new 512 resolution version?

### ISSUE THREAD

**Issue by AlonDan** [neuralchen/SimSwap#163]
For people who use Windows and Anaconda:

1. Should we overwrite the old SimSwap checkpoints?
2. Rename the checkpoint in our python command?

I'm not sure, if we need to make a new enveronment and there are also other installation steps?

Please explain or add a step-by-step guide to help others, thanks ahead!

**Comment by lhucklen** [comment]
Bump this

**Comment by DvST8x** [comment]
Follow the directions they posted, it took under a minute to update mine and to get it working. 

1) Download the latest version of SimSwap, copy it over your current install.
2) From the **Preparation** directions https://github.com/neuralchen/SimSwap/blob/main/docs/guidance/preparation.md
Download the 512.zip and unzip to the checkpoints folder.
3) From the **Usage** directions https://github.com/neuralchen/SimSwap/blob/main/docs/guidance/usage.md
Add **--crop_size 512** to your command line. 

Done, if you like the old way better change **--crop_size 512**  back to **--crop_size 224**

**Comment by bmc84** [comment]
> Follow the directions they posted, it took under a minute to update mine to get it working.
> 
>     1. Download the latest version of SimSwap, copy it over your current install. (Create a backup before copy if you are worried).
> 
>     2. From the **Preparation** directions https://github.com/neuralchen/SimSwap/blob/main/docs/guidance/preparation.md
>        Download the 512.zip and unzip to the checkpoints folder.
> 
>     3. From the **Usage** directions https://github.com/neuralchen/SimSwap/blob/main/docs/guidance/usage.md
>        Add `--crop_size 512` to your command line.
> 
> 
> Done, if you like the old way better change `--crop_size 512` back to `--crop_size 224`

Agreed. Couldn't have been easier. Not sure why people are creating new environments when they have a working version..

**Comment by NNNNAI** [comment]
Hi, you dont need to create a new environment, the old one should be fine. And you dont need to  overwrite the old SimSwap checkpoints, just download the [512.zip](https://github.com/neuralchen/SimSwap/releases/download/512_beta/512.zip) and unzip it in ./checkpoints. And follow the instruction in section "About using Simswap 512 (beta version)" in https://github.com/neuralchen/SimSwap/blob/main/docs/guidance/usage.md can do the magic with Simswap 512.

**Comment by instant-high** [comment]
You can use the exsisting environment.
I've just downloaded the new version to another directory.

e.g
old version : conda activate simswap -> cd c:\simswap -> then run the commandline
new version: conda activate simswap -> cd c:\simswap512 -> then run the commandline
